ArticleService.js

```diff
@@ -0,0 +1,188 @@
+class Article {
+
+    constructor(title, comment, writer, likeCount) {

```

Comment:
보통 최초 생성된 아티클은 likeCount가 0이지 않을까요? 수정하면 다음과 같이 작성할 수 있을 것 같습니다.

```suggestion
    constructor(title, comment, writer) {
        this.title = title;
        this.content = comment;
        this.writer = writer;
        this.likeCount = 0;
        this.createdAt = new Date();
    }
```